README.MD

```diff
@@ -26,6 +26,7 @@ It's always been sad to see a service shutdown, or hardware stop renew, here is
 - [Flowing Data](http://flowingdata.com/category/self-surveillance/) - Self surveillance and quantification news.
 - [Biohack Stack](https://biohackstack.com/) - Biohacker community with "stacks" of famous scientists.
 - [Wearables Database](http://vandrico.com/wearables/) - Awesome list of wearables available per body part.
+- [Database of Health Research related APIs](http://www.health2api.com/?category_name=api&category__and%5B%5D=7854&category__and%5B%5D=0&category__and%5B%5D=0&posts_per_page=50&submit=filter) - More for researchers.
```

Comment:
Maybe it would be better to strip all the query from the URL.

Reply:
That leads to a list of reaserch stufff not QS.

Reply:
Your query seems to result in 5 entries one of which is "Restaurant health inspection scores". I'm not sure if the list is really that relevant for QS.

Reply:
Its great for researchers and developers. All the food apps could really benefit from informing their users of health inspection scores. Same with the supermarket API.

Reply:
A "For developers" sections could be a thing.

Reply:
I agree that it's great for researchers and developers, but I fail to see how it's relevant to the topic at hand.

Health inspection scores are important, but bare not relation at all with the Quantified Self movement. In the same way, Supermarket API may be great for the supermarket industry, but I can't say the same for QS.